<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>617</width>
    <height>420</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/no prefix/resources/4chan.png</normaloff>:/no prefix/resources/4chan.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fixedsys</family>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:none</string>
   </property>
   <property name="text">
    <string>HELLO!!Welcome to Quizzers!!</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>501</width>
     <height>171</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Display Semibold</family>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:none</string>
   </property>
   <property name="text">
    <string>Test your knowledge, challenge friends, and climb the leaderboards!

Ready to dive into thousands of fun quizzes across trivia, science, history, and more? Join the excitement now.

New here? 

Back for more? </string>
   </property>
  </widget>
  <widget class="QPushButton" name="admin_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:none</string>
   </property>
   <property name="text">
    <string>I'm Admin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="user_button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>330</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:none</string>
   </property>
   <property name="text">
    <string>I'm User</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signup_button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:none</string>
   </property>
   <property name="text">
    <string>Sign Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:none</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>admin_button</zorder>
  <zorder>user_button</zorder>
  <zorder>signup_button</zorder>
  <zorder>login_button</zorder>
 </widget>
 <resources>
  <include location="../resources/logo.qrc"/>
 </resources>
 <connections/>
</ui>
